DuckDuckGo/Localizable.xcstrings

```diff
@@ -8344,44 +8344,44 @@
         },
         "es" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "The following Bitwarden versions are incompatible with DuckDuckGo: v2024.3.0, v2024.3.2, v2024.4.0, v2024.4.1. Please update to a newer version by following these steps:"
+            "state" : "translated",
+            "value" : "The following Bitwarden versions are incompatible with DuckDuckGo: v2024.10.0, v2024.10.1, v2024.10.2. Please downgrade to an older version by following these steps:"
```

Comment:
I am not sure about this, can you check with Tom?

Reply:
Looks correct, these were update on Nov 1 in 4de7c6c